composer.json

```diff
@@ -29,7 +29,8 @@
     },
     "require-dev": {
 
-        "phing/phing": "2.14.0"
+        "phing/phing": "2.14.0",
```

Comment:
We do not touch the vendor directory (or packages in it) manually. We use phing to remove unneeded directories/files (Scott came up with this solution in the past), which uses the setting in the following script to remove dirs/files from the packages:
https://github.com/openemr/openemr/blob/master/build.xml
(the goal here is to have it so that in the distant future we can support an automated build process)

So phing is just one time use in the steps in above wiki section to remove stuff from vendor. Then per the steps we manually remove phing since it doesn't belong in the codebase. This is why our way of using composer may not go very well with bringing in other developer type packages (like phpunit) and prob better to just to keep that separated (ie. individual developer sets that up separately outside of composer).